designer/app.py

```diff
@@ -760,12 +761,8 @@ def _perform_new(self, *args):
 
         self.close_popup()
 
-        new_proj_dir = os.path.join(get_config_dir(),
-                                    constants.NEW_PROJECT_DIR_NAME)
-        if os.path.exists(new_proj_dir):
-            shutil.rmtree(new_proj_dir)
-
-        os.mkdir(new_proj_dir)
+        new_proj_dir = tempfile.mkdtemp(prefix='designer_',
+                                        dir=get_config_dir())
```

Comment:
http://superuser.com/questions/296824/when-is-a-windows-users-temp-directory-cleaned-out